pkg/testutil/object.go

```diff
@@ -16,6 +16,10 @@ import (
 	"sigs.k8s.io/cli-utils/pkg/object/dependson"
 )
 
+// OwningInventoryKey is the annotation key indicating the inventory owning an object.
+// This is a copy of inventory.OwningInventoryKey to avoid dependency cycle.
+const OwningInventoryKey = "config.k8s.io/owning-inventory"
```

Comment:
It occurs to me that we could avoid this const copy if we injected it into the mutators that use it. Then we could change them to AddAnnotation and DeleteAnnotation. 

Is it worth doing in this PR or a follow up?

Reply:
This is already a big PR, so do a follow-up if possible.